Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -346,7 +346,7 @@ public class Fuzzer {
     func collectRuntimeTypes(_ program: Program) {
```

Comment:
I think eventually it would be good to collect some statistics on how frequently type collection fails (the percentage of failed collections vs successful collections could then e.g. just be printed with one of the warning messages logged below). Doesn't have to be in this PR though.

Reply:
well there should be failures only in very specific cases, so this number maybe makes sense only for time outs